### ISSUE TITLE
No information available on Structure or subitems

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#7448]
Here there is "No information available" when instead there should be some link to https://wiki.openstreetmap.org/wiki/Key:bridge etc.
![10661-2](https://user-images.githubusercontent.com/1230959/77272199-961fc400-6ceb-11ea-8a08-cb99eb5bef7f.jpg)
Therefore each subitem needs an "i" link.

**Comment by sun-geo** [comment]
Yes, adding "i" to each line could be a solution, or creating a top level structure wiki article page "structure" which contains some links to different tag/key wiki pages. Maybe a little bit similar to this page:
https://wiki.openstreetmap.org/wiki/Note

**Comment by quincylvania** [comment]
> or creating a top level structure wiki article page "structure" which contains some links to different tag/key wiki pages

I'd prefer this to adding five more buttons.

**Comment by jidanni** [comment]
But wait, with the five buttons the user could get to the info of each item directly!
